src/anonymous_store.h

```diff
@@ -405,9 +418,7 @@ AnonymousResult AnonymousStore<key_type>::put_external(
     } else if (ctrm.source() == zsearch::CERTIFICATE_SOURCE_MOZILLA_SALESFORCE) {
         zsearch::Certificate* cert = rec->mutable_certificate();
         zsearch::CertificateAudit* audit = cert->mutable_audit();
-        zsearch::MozillaSalesForceStatus* nss = audit->mutable_mozilla();
-        nss->set_current_in(true);
-        nss->set_was_in(true);
+        audit->mutable_mozilla()->CopyFrom(ctrm.nss_status());
```

Comment:
I'm not familiar with what's in an ExternalCertificate, but I'll assume that since this compiles it's correct.